proxy/outprocess/src/module_loaders/outprocess_module.c

```diff
@@ -38,6 +38,7 @@ typedef struct OUTPROCESS_HANDLE_DATA_TAG
 	STRING_HANDLE module_args;
 	OUTPROCESS_MODULE_LIFECYCLE lifecyle_model;
 	BROKER_HANDLE broker;
+	int default_wait;
```

Comment:
And check the type. time_t, uint or size_t would be more appropriate.

---
In reply to: [106021710](https://github.com/Azure/azure-iot-gateway-sdk/pull/160#discussion_r106021710) [](ancestors = 106021710)